Src/Newtonsoft.Json.UnityConverters/EnumerableVectorConverter.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using UnityEngine;
+#if (PORTABLE)
+using System.Reflection;
+#endif
+
+namespace Newtonsoft.Json.UnityConverters
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class EnumerableVectorConverter<T>: JsonConverter
+    {
+        private static readonly VectorConverter VectorConverter = new VectorConverter();
+
+#if(PORTABLE)
+        private static readonly TypeInfo V2TypeInfo = typeof(IEnumerable<Vector2>).GetTypeInfo();
+        private static readonly TypeInfo V3TypeInfo = typeof(IEnumerable<Vector3>).GetTypeInfo();
+        private static readonly TypeInfo V4TypeInfo = typeof(IEnumerable<Vector4>).GetTypeInfo();
+#endif
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="writer"></param>
+        /// <param name="value"></param>
+        /// <param name="serializer"></param>
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (value == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4846771)